apps/chat-e2e/src/testData/conversationHistory/conversationBuilder.ts

```diff
@@ -10,21 +10,19 @@ import {
   Message,
   Replay,
 } from '@/chat/types/chat';
-import { constructPath } from '@/chat/utils/app/file';
-import { getRootId } from '@/chat/utils/app/id';
-import { ApiKeys } from '@/chat/utils/server/api';
 import { ModelsUtil } from '@/src/utils';
 import { v4 as uuidv4 } from 'uuid';
 
+export interface TestConversation extends Omit<Conversation, 'folderId'> {
+  folderId?: string | undefined;
+}
+
 export class ConversationBuilder {
-  private conversation: Conversation;
+  private conversation: TestConversation;
 
   constructor() {
     this.conversation = {
-      id: constructPath(
-        getRootId({ apiKey: ApiKeys.Conversations }),
-        DEFAULT_CONVERSATION_NAME,
-      ),
+      id: uuidv4(),
```

Comment:
Are you sure that creating conversation with uuid is good idea? We are having some logic where we are parsing some data from id

Reply:
conversation is not created here, this is just an object. I left here uuid for compatibility with local storage mode. If stateful API is used, conversation id is adjusted according to 'path/model__name' format. See itemApiHelper.ts class